lld/include/lld/Common/ErrorHandler.h

```diff
@@ -152,6 +152,21 @@ void message(const Twine &msg, llvm::raw_ostream &s = outs());
 void warn(const Twine &msg);
 uint64_t errorCount();
```

Comment:
Is the plan to replace these error methods with `elf::Warn()`, `elf::Err()`, ..., etc? If so can we add a doc to point users to these new methods?

Reply:
For ELF, the intention is yes. Perhaps this "deprecation" notice can be added once another port decides to follow ELF?